### DISCUSSION TITLE
Can't submit my GitHub Student Developer Pack application

### DISCUSSION THREAD

**Discussion by M1sttt** [community/community#72646] [GitHub Education]
### Hi everyone,

I'm trying to submit my application.
I have a school-issued email and it does recognize it so I just need to upload my academic proof.
I upload the jpg file and it works but when I try to submit it just says `Unable to submit your application. Please try again` with no further explanation.
Can someone relate or have a solution for this?
Thanks.

![image](https://github.com/community/community/assets/66747892/b2801c0d-2df6-4e7d-ac08-0fed2042fb32)

**Comment by ktauchathuranga** [comment]
i think it would be better if you opened a support request via following link:
[GitHub Support](https://support.github.com/request)

**Comment by Kr1s7on** [comment]
Same bruh
